pkg/sql/create_table.go

```diff
@@ -1307,14 +1307,6 @@ func makeCheckConstraint(
 	semaCtx *tree.SemaContext,
 	evalCtx *tree.EvalContext,
 ) (*sqlbase.TableDescriptor_CheckConstraint, error) {
-	// CHECK expressions seem to vary across databases. Wikipedia's entry on
-	// Check_constraint (https://en.wikipedia.org/wiki/Check_constraint) says
-	// that if the constraint refers to a single column only, it is possible to
-	// specify the constraint as part of the column definition. Postgres allows
-	// specifying them anywhere about any columns, but it moves all constraints to
-	// the table level (i.e., columns never have a check constraint themselves). We
```

Comment:
Hmm, I kind of like the rest of this comment. Would it be totally crazy to just modify the end to explain what we actually do?

Reply:
Yeah, could just change the last sentence to "We do the same."